README.md

```diff
@@ -2,3 +2,30 @@
 
 [![Join the chat at https://gitter.im/hitraffic/mobile](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hitraffic/mobile?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 A hybrid mobile app powered by the Ionic Framework.  Utilizes the hitraffic API to provide traffic incident reports.
+
+---
+
+Installation
+-----------------------------
+
+1. Install Cordova by running `sudo npm install -g cordova`
```

Comment:
I've thought long and hard about this and have decided that "sudo" should be in parentheses because it may not be required.

:pray: Please don't hate me for being nitpicky :pray: